runtime/doc/supported_platforms.txt

```diff
@@ -0,0 +1,55 @@
+*supported_platforms.txt*          Nvim
```

Comment:
This is over-specific for a single help page (and was also over-specific for a single markdown page).

Maybe `support.txt` makes sense? Then we can later add sections for how to Install, how to Build, where to get help?

Reply:
I'd prefer `support.txt` (as a help page, we have tags that can take us to a subsection directly -- we just need to use a deep link in the README then).

`intro.txt` is... weird.